armi/bookkeeping/tests/test_databaseInterface.py

```diff
@@ -265,7 +266,14 @@ def test_readWritten(self):
 
                 for c1, c2 in zip(sorted(b1), sorted(b2)):
                     self.assertEqual(c1.name, c2.name)
-                    assert_equal(c1.spatialLocator.indices, c2.spatialLocator.indices)
+                    if isinstance(c1.spatialLocator, grids.MultiIndexLocation):
+                        assert_equal(
+                            c1.spatialLocator.allIndices, c2.spatialLocator.allIndices
+                        )
+                    else:
+                        assert_equal(
+                            c1.spatialLocator.indices, c2.spatialLocator.indices
+                        )
```

Comment:
didn't  want to override `__eq__` on `spatialLocator` itself?

Reply:
There is an __eq__() impl, but it includes the grid itself...